src/NUnitConsole/nunit3-console.tests/CosoleRunnerTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Xml;
+using NUnit.Common;
+using NUnit.Engine;
+using NUnit.Engine.Extensibility;
+using NUnit.Engine.Services;
+using NUnit.Framework;
+
+namespace NUnit.ConsoleRunner.Tests
+{
+    class CosoleRunnerTests
+    {
+        [Test]
+        public void XXX()
```

Comment:
Could you rename this to something more meaningful? Maybe `ThrowsNUnitEngineExceptionWhenTestResultsAreNotWriteable`?